src/pages/management/configuration-api/v3.6/index.mdx

```diff
@@ -3057,7 +3057,7 @@ curl -X POST \
 
 Enables the webhooks [registered](#register-webhook) by a given **Client Id** (application). The webhooks will be installed for the license associated with the access token used in the request.
 
-Webhooks will be automaticall enabled if you [register them via Developer Console](/getting-started/app-guides/chat-webhooks/).
```

Comment:
fixed typo from #platform-docs (haven't updated preview for that change as it's tiny)